src/javascript/class-scanner.ts

```diff
@@ -211,6 +213,7 @@ export class ClassScanner implements JavaScriptScanner {
       astNode,
       properties,
       methods,
+      staticMethods,
```

Comment:
Ah I see. I guess it would be nice in terms of organization if there was an element scanner that was separate to the class function, and a minimal shared function that could distinguish between the two.